serverless.yml

```diff
@@ -15,7 +15,7 @@ service: library-management-slack-bot
 
 # You can pin your service to only deploy with a specific Serverless version
 # Check out our docs for more details
-frameworkVersion: "=1.38.0"
+frameworkVersion: ">=1.38.0"
```

Comment:
I pinned to `1.38.0` because `1.39.0` had me running into this bug. https://github.com/serverless/serverless/issues/4720